recipes/tsv-utils/build.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+set -x
+
+make DCOMPILER=ldc2 LDC_LTO_RUNTIME=1 LDC_PGO=2
```

Comment:
I think if you remove `LDC_PGO` you might not have problems wtih `gold` anymore.

Reply:
Sadly didn't help
```
Error: The LLVMgold.so plugin (needed for LTO) was not found. You can specify its path with -flto-binary=<file>.
```
https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=422923&view=logs&j=6f142865-96c3-535c-b7ea-873d86b887bd&t=22b0682d-ab9e-55d7-9c79-49f3c3ba4823&l=949

Reply:
For mac the error type is different:
```
make -C csv2tsv 
make[1]: Entering directory '$SRC_DIR/csv2tsv'
dyld: lazy symbol binding failed: Symbol not found: _dyld_enumerate_tlv_storage
  Referenced from: /Users/runner/mambaforge/conda-bld/tsv-utils_1638701452887/_build_env/bin/ldc2
  Expected in: /usr/lib/libSystem.B.dylib

dyld: Symbol not found: _dyld_enumerate_tlv_storage
  Referenced from: /Users/runner/mambaforge/conda-bld/tsv-utils_1638701452887/_build_env/bin/ldc2
  Expected in: /usr/lib/libSystem.B.dylib
```
https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=422923&view=logs&j=e35d4f76-8ff2-5536-d795-df91e63eb9f7&t=fa7b4b17-b6ff-5c9c-8cfc-15f888c92310&l=1682

Are binaries not accepted at conda-forge? That was working already.

If we can't get it to work here and binaries are not accepted, bioconda accepts binaries. Maybe that's more suitable then?